<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferFrame</class>
 <widget class="QFrame" name="TransferFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Box</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Pay To:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="m_addressEdit">
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_addressBookButton">
       <property name="toolTip">
        <string>Select recipient from Address Book</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/add_recepient</normaloff>:/icons/add_recepient</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_pasteButton">
       <property name="toolTip">
        <string>Paste from clipboard</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/paste</normaloff>:/icons/paste</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_removeButton">
       <property name="toolTip">
        <string>Remove recepient</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Amount:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="m_amountSpin">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="suffix">
      <string notr="true"/>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>99999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Save to Contacts as:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_labelEdit">
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>Enter a label for this address to add it to your address book</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_addressEdit</tabstop>
  <tabstop>m_amountSpin</tabstop>
  <tabstop>m_labelEdit</tabstop>
  <tabstop>m_addressBookButton</tabstop>
  <tabstop>m_pasteButton</tabstop>
  <tabstop>m_removeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_removeButton</sender>
   <signal>clicked()</signal>
   <receiver>TransferFrame</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addressBookButton</sender>
   <signal>clicked()</signal>
   <receiver>TransferFrame</receiver>
   <slot>addressBookClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>786</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pasteButton</sender>
   <signal>clicked()</signal>
   <receiver>TransferFrame</receiver>
   <slot>pasteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>824</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_amountSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransferFrame</receiver>
   <slot>amountValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_removeButton</sender>
   <signal>clicked()</signal>
   <receiver>TransferFrame</receiver>
   <slot>amountValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>861</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addressEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>TransferFrame</receiver>
   <slot>addressEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addressBookClicked()</slot>
  <slot>pasteClicked()</slot>
  <slot>addressEdited(QString)</slot>
 </slots>
</ui>
